tests/cli/upload/conftest.py

```diff
@@ -163,10 +241,33 @@ def files(self, **kwargs):
             return MockFile(empty_first=True)
         return MockFile()
 
+    def get_files(self, bundle, tags, version="1.0"):
+        """docstring for get_files"""
+        return self._files
+
+    def add_file(self, file, version_obj, tag_name, to_archive=False):
+        """docstring for add_file"""
```

Comment:
Same here - I seems auto generated or is this by intent.

Reply:
No I'll fix that